### PR TITLE
Adding AppVeyor integration for Rocket.Chat

### PR DESCRIPTION
**Pull request by crazy-max**
Here is my contrib linked to issue #471
Uptime Robot integration doc has been updated too with advanced notifications.
Original PR #472 since refactoring.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 .DS_Store
-
```
Comment (MartinSchoeler) [inline .gitignore]: Why did you Change the `.gitignore` file?

Comment (crazy-max) [reply .gitignore]: To exclude *.iml file (specific to Intellij IDE), is that a problem?

Comment (MartinSchoeler) [reply .gitignore]: I don't think there is need to add a new exclude to the gitignore file for now.

Comment (crazy-max) [reply .gitignore]: Ok @MartinSchoeler, i have reverted the commit

**Review by MartinSchoeler — APPROVED** [review]
Thanks for the Contribution!
